frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import {
+  confirmAction,
+  createStorageClassButton,
+  emptyStateBody,
+  storagePoolDropdown,
+  storagePool,
+} from '../views/multiple-pool';
+
+describe('Test Ceph pool creation', () => {
+  // Pool var
+  const poolName: string = 'example.pool';
+  const replicaCount: string = '2';
+  const RBDProvisioner: string = 'openshift-storage.rbd.csi.ceph.com';
```

Comment:
Let's move it outside of the describe block. This would confuse people to think that these values are somewhat dependent on the runtime.